packages/mds-event-server/index.ts

```diff
@@ -39,7 +39,8 @@ export const EventServer = <TData, TResult>(
       return res.status(200).send({ result })
     } catch (error) /* istanbul ignore next */ {
       await logger.error('Cloud Event', error, { method, headers, body })
-      return res.status(500).send({ error })
+      // Must return healthy status to KNE to prevent infinite retry looping
+      return res.status(200).send({ error })
```

Comment:
I wonder if there is another `2xx` code that we could use that KNE would still like, but that could possibly be used to differentiate if logged, etc.

Reply:
Changed to 500 for malformed CEs and 202 for all other errors